pke/kyber/internal/common/poly_test.go

```diff
@@ -1,15 +1,16 @@
 package common
 
 import (
-	cryptoRand "crypto/rand"
+	"crypto/rand"
 	"fmt"
-	mathRand "math/rand"
 	"testing"
 )
 
 func (p *Poly) RandAbsLe9Q() {
+	r := randSliceUint32(uint(N))
+	max := 9 * uint32(Q)
 	for i := 0; i < N; i++ {
-		p[i] = int16(mathRand.Intn(18*int(Q) - 9*int(Q)))
+		p[i] = int16(int32(r[i] % max))
```

Comment:
This change looks equivalent to me. However, the way the original code was written is weird. (Why `18 * int(Q) - 9 * int(Q)` instead of just `9 * int(Q)`?) This makes me think it might be worth double checking that this is the correct range.

Reply:
@bwesterb could you please give us some light here?

Reply:
let's move this change forward, and do changes in other PR if bwesterb thinks it's needed.

Reply:
The intended code was `int16(mathRand.Intn(18*int(Q)) - mathRand.Intn(9*int(Q)))`.

Reply:
Thanks Bas, I will fix this on #240